data.json

```diff
@@ -965,4 +965,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/49101362/120364787-7c501880-c2c2-11eb-8f18-0f0ef10c4439.png",
     "issueNumber": 539
   }
```

Comment:
```suggestion
  },
```
there is a comma missing

Reply:
Sorry! changed now #545

Reply:
But it doesn't contain your entry

Reply:
No problem. For this issue you don't have to make a new Pull Request. I think it also wouldn't working this way 🤔 .

You can adjust the issues in this pull request in this branch https://github.com/skjha1/awesome-github-profiles/tree/patch-1 if you create another branch this will open a new PR which is not connected to this one.

So what to do now :)

You can add the missing comma in the patch-1 branch of your fork or you can simply commit the suggestion here in the PR by hitting the "Commit Suggestion" Button above.
![image](https://user-images.githubusercontent.com/30869493/120445682-272b0a00-c389-11eb-8df0-f1407c87a99d.png)

Both ways will add a commit into this Pull Request and then I think we are free to go to merge it :)

If there are some more questions please let us know 🚀

Reply:
> But it doesn't contain your entry

Fast as always :D